src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulk/scan/transformation/FormAToCaseTransformerTest.java

```diff
@@ -396,8 +398,19 @@ public void shouldTransformAnyAuthorisationSignedToYes() {
     public void shouldTransformScannedDocuments() {
         List<InputScannedDoc> scannedDocuments = new ArrayList<>();
         scannedDocuments.add(InputScannedDoc.builder().subtype("D81").document(new InputScannedDocUrl("http://url/d81-1", "http://binUrl/d81-1/binary", "d81-1.pdf")).build());
-        scannedDocuments.add(InputScannedDoc.builder().subtype("FormA").document(new InputScannedDocUrl("http://url/formA", "http://binUrl/formA/binary", "formA.pdf")).build());
         scannedDocuments.add(InputScannedDoc.builder().subtype("D81").document(new InputScannedDocUrl("http://url/d81-2", "http://binUrl/d81-2/binary", "d81-2.pdf")).build());
+        scannedDocuments.add(createDoc("FormA"));
```

Comment:
not "formA"?

Reply:
This is what's specified in the SIT form.